BeeJet.Tests/BeeJet.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
De rest van de projects zit nog op .net 6.0; weet niet of het verstandig is de tests een versie hoger te draaien. Dat gaat denk ik gedoe opleveren bij de build + test action.

Reply:
ah ja dat hebben we recent weer teruggezet naar 6.
kan ik nog wel fixen.